src/gc/CMakeLists.txt

```diff
@@ -51,6 +89,8 @@ else()
   set (GC_LINK_LIBRARIES)
 endif(WIN32)
 
+list(APPEND GC_SOURCES ${GC_HEADERS})
```

Comment:
I wonder if it would make sense to add  the following too to keep headers separated from source files in Visual Studio.
```cmake
source_group("Headers" FILES ${GC_HEADERS})
```

Reply:
All sources which are included into vcxproj by CMake are separated by filters, headers will always be placed under `headers` filter while `sources` will be under `sources` filter. Fortunately for us CMake does it automatically by simply creating filters in .vcxproj.

For example clrjit.vcxproj is divided into following groups of files:

![clrjit vcxproj](https://user-images.githubusercontent.com/801959/45623756-aa488c80-ba88-11e8-9d29-12c0918d979e.png)

Reply:
Ok, great!